Dockerfile

```diff
@@ -8,6 +8,8 @@ ENV LC_ALL en_US.UTF-8
 # Fix multilib issues when installing openssl-devel.
 RUN yum install -y --enablerepo=centosplus libselinux-devel && yum clean all
 
+RUN yum install -y supervisor bash-completion && yum clean all
```

Comment:
you can just move this down to line 17 along with all the other package installs